ada/ast.py

```diff
@@ -14000,6 +14051,25 @@ class Identifier(BaseId):
         )
     )
 
+    @langkit_property()
+    def is_constant():
+        return Or(
+            Entity.referenced_decl.is_constant_object,
+
+            # An instance of a protected unit is constant within
```

Comment:
The comment speaks about the protected unit, but here you're not checking the unit but the variables defined inside the unit, so the comment should be adjusted to talk about what you're really doing